### PR TITLE
Tkach/header

### PR DESCRIPTION
**Pull request by Tkach-Illia**

### CODE REVIEW

File: front-end/src/components/UI/LoginButton/MyButton/MyButton.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from "../LoginButton.module.css";
+
+const MyButton = ({text, callback,style}) => {
```
Comment (azazak123) [inline front-end/src/components/UI/LoginButton/MyButton/MyButton.jsx]: Does it make sense to rename this to something like `ToggleButton`?

File: front-end/src/components/UI/LoginButton/MyForm/MyForm.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import styles from "./MyForm.module.css";
+import LogPage from "../Variants/LogPage";
+import SignPage from "../Variants/SignPage";
+
+const MyForm = ({visibleLogIn, setVisibleLogIn, visibleSignIn, setVisibleSignIn}) => {
```
Comment (azazak123) [inline front-end/src/components/UI/LoginButton/MyForm/MyForm.jsx]: Let's rename this to `LoginForm`

File: front-end/src/components/UI/LoginButton/MyForm/MyForm.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    function setVisible(bool){
+        setVisibleLogIn(bool)
+        setVisibleSignIn(bool)
+    }
+
+    function sendData(data){
+        console.log(data)
+        setVisible(false)
+        setData({login: '',email:'', password: ''});
+    }
```
Comment (azazak123) [inline front-end/src/components/UI/LoginButton/MyForm/MyForm.jsx]: Let's remake this functions as lambda (`const setVisible = (status) => {...}`)

File: front-end/src/components/UI/LoginButton/Variants/LogPage.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import cl from './LogPage.module.css'
+
+const LogPage = ({data, setData, callback}) => {
```
Comment (azazak123) [inline front-end/src/components/UI/LoginButton/Variants/LogPage.jsx]: Let's use full name (`LoginPage`) and rename `data` to the `userData`

File: front-end/src/components/UI/LoginButton/Variants/SignPage.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cl from './LogPage.module.css'
+
+const SignPage = ({data, setData, callback}) => {
```
Comment (azazak123) [inline front-end/src/components/UI/LoginButton/Variants/SignPage.jsx]: Same with data (`data` => `userData`)

**Review by azazak123 — COMMENTED** [review]
In general good, but couple comments. Also, I would be grateful if you could use linter and formatter for code

File: front-end/src/components/UI/LoginButton/MyForm/MyForm.jsx (hunk 3 above)
Comment (Tkach-Illia) [reply front-end/src/components/UI/LoginButton/MyForm/MyForm.jsx:23]: Not a good idea. This function use more then one time

**Review by azazak123 — APPROVED** [review]

**Pull request merged**